modules/oas-page-builder/tests/unit/services/pageBuilder.test.ts

```diff
@@ -192,7 +192,7 @@ describe('pageBuilder', () => {
 
     await buildOpenAPIPages(testEntries, testOptions);
 
-    expect(mockExecute).toBeCalledTimes(testEntries.length * 2);
+    expect(mockExecute).toBeCalledTimes(4);
```

Comment:
Given that we specify `toBeCalledWith` for each invocation, I'm fine with hardcoding the number of calls now that the logic isn't as straightforwards.

Open to any other equations that determine that intended number of calls though.